tests/js/web/DropZoneBodyClassManagerTest.js

```diff
@@ -0,0 +1,23 @@
+import DropZoneBodyClassManager from '../../../js/DropZone/DropZoneBodyClassManager';
+
+describe('DropZoneBodyClassManager', () => {
+    let $body;
+    let classManager;
+
+    beforeEach(() => {
+        $body = $('<div class="foo"></div>');
+        classManager = new DropZoneBodyClassManager();
+    });
+
+    describe('update()', () => {
+        it('should update the body class with the new state', () => {
+            classManager.update($body[0], ['bar', 'baz']);
+            expect($body.attr('class')).to.equal('foo bar baz');
```

Comment:
Personal preference, but I find it nice to have a blank line between the sections:
```javascript
// Arrange (although I realise there is no "arrange" section is this particular test)
this.thing.configure(...);

// Act
this.thing.doSomething();

// Assert
expect(this.otherThing).to.have.been.calledOnce;
```